Formula/k3d.rb

```diff
@@ -19,14 +20,17 @@ class K3d < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "e117dd2b6ebf56a0def38e790920f5c9ac9bbdbfe1abb617c933f46db8770f40"
   end
 
+  depends_on "gnu-sed" => :build
   depends_on "go" => :build
 
+  # curl and sed used to fetch/parse latest stable K3s version
+  uses_from_macos "curl" => :build
```

Comment:
`curl` is always installed (Homebrew doesn't work if you don't have a pre-existing `curl`), so unless you need Homebrew `curl` on Linux specifically, you don't need to declare this.

Reply:
Good point, I'll remove this then.. was going to argue about it, but if curl is a brew requirement anyway, it doesn't make sense for real (it's in a few other formulae as well).

Reply:
:heavy_check_mark:

Reply:
Other formulae use it because they require a new version of `curl` (i.e. the minimum required by Homebrew doesn't suffice). At least, that's the idea. There are probably some that we can get rid of.